spec/models/atmosphere/virtual_machine_spec.rb

```diff
@@ -185,7 +185,10 @@ def delete_with_success_when_exception(e)
   end
 
   context 'DNAT' do
-    let(:vm) { create(:virtual_machine, ip: priv_ip, appliances: [appliance]) }
+    let(:vm) { create(:virtual_machine,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.